### PR TITLE
Overview searches

### PR DESCRIPTION
**Pull request by kevinemoore**
## Description
Adding two new actions to the search lambda: images (returns a list of images from the bucket) and other (returns a list of other pre-viewable files).

### CODE REVIEW

File: lambdas/search/index.py [hunk 1]
```diff
@@ -50,6 +73,26 @@ def lambda_handler(request):
...
         _source = []
+    elif action == 'images':
+        body = {
+            'query': {
+                'terms': {
+                    "ext": IMG_EXTS
+                }
+            }
+        }
+        size = 10
```
Comment (akarve) [inline lambdas/search/index.py]: I think we should request 100 of these, e.g. https://open.quiltdata.com/b/nasanex. @nl0 can yo confirm they are lazy loaded?

Comment (akarve) [inline lambdas/search/index.py]: both `size = ` values should be set by GLOBALS, e.g. `NUM_PREVIEW_IMAGES`, `NUM_PREVIEW_FILES`

File: lambdas/search/index.py [hunk 2]
```diff
@@ -50,6 +73,26 @@ def lambda_handler(request):
...
+        _source = []
+    elif action == 'other':
+        body = {
+            'query': {
+                'terms': {
+                    "ext": OTHER_EXTS
```
Comment (akarve) [inline lambdas/search/index.py]: close but not quite what we want. we really should have a bias towards `readme.*` files

Comment (akarve) [reply lambdas/search/index.py]: i guess in the general case that implies a full scan; can we at least look for `ROOT/readme.*`?

File: lambdas/search/index.py (hunk 1 above)
Comment (nl0) [reply lambdas/search/index.py]: confirming

File: lambdas/search/index.py (hunk 2 above)
Comment (kevinemoore) [reply lambdas/search/index.py]: A simple search for "README" works pretty well to grab just READMEs. Do we need anything fancier than that?

**Pull request merged**